uk.co.compucorp.civicrm.hrleaveandabsences/js/angular/src/leave-absences/shared/controllers/leave-request-popup-controller.js

```diff
@@ -601,17 +641,18 @@ define([
        * @return {Promise}
        */
       function initLeaveRequest() {
+        var attributes, request;
```

Comment:
```javascript
function initLeaveRequest() {
  var attributes;

  //if set indicates that leaverequest is either being managed or edited
  if (directiveOptions.leaveRequest) {
    //get a clone so that it is not the same reference as passed from callee
    attributes = _.cloneDeep(directiveOptions.leaveRequest.attributes());
  } else {
    attributes = { contact_id: directiveOptions.contactId };
  }
  vm.leaveRequest = vm.isleaveType('sick') ? SicknessRequestInstance.init(attributes) : LeaveRequestInstance.init(attributes);
  return $q.resolve(vm.leaveRequest);
}
```